commands/diff/__tests__/__snapshots__/diff-command.test.js.snap

```diff
@@ -44,10 +44,10 @@ index SHA..SHA 100644
 `;
 
 exports[`DiffCommand should diff packages from the most recent tag 1`] = `
-diff --git a/packages/package-1/package.json b/packages/package-1/package.json
+diff --git c/packages/package-1/package.json w/packages/package-1/package.json
 index SHA..SHA 100644
---- a/packages/package-1/package.json
-+++ b/packages/package-1/package.json
+--- c/packages/package-1/package.json
++++ w/packages/package-1/package.json
```

Comment:
I'm not sure why these snapshots needed to be updated. Are you using a non-English localization of git?

Reply:
I'm on an english localization of git, just my OSX is in French. I revert my package-lock, ran `npm ci` the only diff is the new test. I have the same problems with snapshots.